### PR TITLE
refactor(sentry): use crashpad as sentry backend

### PR DESCRIPTION
**Pull request by subotic**
Part of DEV-2988

### CODE REVIEW

File: src/sipi.cpp [hunk 1]
```diff
@@ -1000,10 +995,6 @@ int main(int argc, char *argv[]) {
 
         try {
             std::cout << std::endl << "Ivan was here" << std::endl;
```
Comment (BalduinLandolt) [inline src/sipi.cpp:997]: Is this line still needed? :P

**Review by BalduinLandolt — APPROVED** [review]
I understand very little of what's going on here, but from what I see, it makes sense

File: src/sipi.cpp (hunk 1 above)
Comment (subotic) [reply src/sipi.cpp:997]: I like it ;-) It is also used in some queries in Infra.

Comment (subotic) [reply src/sipi.cpp:997]: Sadly, the only way to figure out if Sipi restarted.

**Pull request merged**